src/bld/java/rife/BldBuild.java

```diff
@@ -132,11 +132,9 @@ public void zipBld()
                             f.perms(0755);
                         });
                     });
-                    b.dir("lib", l -> {
-                        l.file("bld-wrapper.jar", f -> f.move(path(tmp, "lib", "bld", "bld-wrapper.jar")));
-                    });
+                    b.dir("lib", l -> l.file("bld-wrapper.jar", f -> f.move(path(tmp, "lib", "bld", "bld-wrapper.jar"))));
                 });
-                t.dir("lib", l -> l.delete());
+                t.dir("lib", DirBuilder::delete);
```

Comment:
This is longer and it breaks the pattern of the other actions, I prefer it to remain `l -> l.delete()`